database/migrations/phinx/20141105085836_form_attributes_relations.php

```diff
@@ -38,7 +38,7 @@ public function up()
         }
 
         // drop the pivot table
-        $this->dropTable('form_groups_form_attributes');
+        $this->table('form_groups_form_attributes')->drop()->save();
```

Comment:
What is the reason behind changes like these? New phinx idioms?

Reply:
Oh yes ... the update phinx lib no longer supports the `dropTable` method.